virttest/qemu_qtree.py

```diff
@@ -342,10 +342,15 @@ def _hook_usb2_disk(node):
                     # qemu commit a5f5429 introduced named GPIOs, So add string
                     # key into line[0] to distinguish GPIOs, and save exact
                     # value into line[1].
+                    # qemu-kvm: gpio-in 24
+                    # qemu-kvm-rhev: gpio-in "" 1
                     elif line[0] == 'gpio-in' or line[0] == 'gpio-out':
                         tmp_field = line[1].split(' ', 1)
-                        line[0] = "%s %s" % (line[0], tmp_field[0])
-                        line[1] = tmp_field[1]
+                        if '""' in line[1]:
```

Comment:
This can not handle the following line:
```
gpio-out "a20" 1
```
I prefer to use this approach:
```python
elif line[0] == 'gpio-in' or line[0] == 'gpio-out':
    if ' ' in line[1]:
        gpio_name, line[1] = line[1].split(' ', 1)
        line[0] += " %s" % gpio_name
```